### PR TITLE
release prep

### PR DESCRIPTION
**Pull request by ritikarawlani**

### CODE REVIEW

File: input/pagecontent/ig_licensing.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (ritikarawlani) [inline input/pagecontent/ig_licensing.md:1]: @costateixeira can we remove this file?

Comment (costateixeira) [reply input/pagecontent/ig_licensing.md:1]: we can. I think this page was added to explain the different considerations for all IGs (not only WHO). we can leave it out for now?

File: input/pagecontent/ig_setup.md [hunk 2]
```diff
@@ -58,4 +58,4 @@ After initializing the IG, it must be configured:
 * Package id: `who.fhir.????` Or `who.fhir.smart.???` `Or who.smart.fhir.?`
```
[the remark this answers is not in the document]
Comment (costateixeira) [reply input/pagecontent/ig_setup.md]: done. the ig setup now points at the ig configuration for more details

[the remark this answers is not in the document]
Comment (costateixeira) [reply input/pagecontent/ig_setup.md]: we can also remove the ig_configuration altogether. what is the best option?

[the remark this answers is not in the document]
Comment (costateixeira) [reply input/pagecontent/ig_setup.md]: @ritikarawlani

File: input/pagecontent/ig_licensing.md (hunk 1 above)
Comment (costateixeira) [reply input/pagecontent/ig_licensing.md:1]: @ritikarawlani

**Pull request merged**